dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.34/V2_34_1__Update_program_rule_action_with_environments_and_evaluation_time_columns_and_default_values.sql

```diff
@@ -0,0 +1,6 @@
+alter table programruleaction add evaluationtime varchar(50);
+alter table programruleaction add environments jsonb;
+
+update programruleaction set evaluationtime = 'ALWAYS' where evaluationtime is null;
+update programruleaction set environments = '["WEB", "ANDROID"]' where environments is null;
```

Comment:
Could be useful to add the "::jsonb" cast behind the array just to make it clear for others this is actually expected to be interpreted as jsonb as well.